lib/utils/setCookie.js

```diff
@@ -18,13 +18,17 @@ module.exports = function setCookie (storage, name, value) {
     local: storage.domain === 'localhost',
     custom: storage.customDomain
   }
+
+  var sameSite = 'None'
+
   var expires = new Date()
   expires.setSeconds(expires.getSeconds() + clone.expires)
 
   // When it's localhost, an IP, or custom domain, set the cookie directly
-  if (is.ip || is.local || is.custom) {
-    clone.domain = is.local ? null : clone.domain
-    cookie.setItem(name, value, expires, clone.path, clone.domain)
+  if (is.local) {
+    cookie.setItem(name, value, expires, clone.path, null)
+  } else if (is.ip || is.custom) {
+    cookie.setItem(name, value, expires, clone.path, clone.domain, true, sameSite)
```

Comment:
IMO, for easier read:
`cookie.setItem(name, value, expires, clone.path, true, 'None')`